xapian-applications/omega/templates/query

```diff
@@ -58,7 +58,9 @@ if (!("autofocus" in document.createElement("input")))
  document.getElementById("omega-autofocus").focus();
 </script>
 <input type=submit value="Search">
-$if{,$include{inc/anyalldropbox},$include{inc/anyallradio}}
+${Todo: Replace radios with search buttons}
+$if{,$include{inc/anyalldropbox},
+<fieldset>$include{inc/anyallradio}</fieldset>}
```

Comment:
If `inc/anyallradio` gets a fieldset, `inc/anyalldropbox` probably should do also. (It's an example of another way of doing things, it won't get used automatically.)

This leaves other input elements outside fieldsets. The search box & submit button probably make sense to wrap as well. And topterms should almost certainly be a fieldset.

You should consider adding a legend to the fieldsets (we can style it away later if necessary).

(You should have put the classnames on this in the same commit. The change is that you're improving this part of the HTML, not that you're adding "fieldset" in.)